docs/installation/linux.md

```diff
@@ -339,6 +339,33 @@ and is a single binary.
 
 ## macOS 10.12
 
+### Installation via Homebrew (preferred) - macOS 10.12
+
+[Homebrew][brew] is the missing package manager for macOS.
+If the `brew` command is not found,
+you need to install Homebrew following [their instructions][brew].
+
+Once you've installed Homebrew, installing PowerShell is easy.
+Just install [Homebrew-Cask][cask] and then you can install PowerShell:
+
+```sh
+brew tap caskroom/cask
+brew cask install powershell
+```
+
+When new versions of PowerShell are released,
+simply update Homebrew's formulae and upgrade PowerShell:
+
+```sh
+brew update
+brew upgrade powershell
```

Comment:
Oh, interesting...thanks for the pointer. Looks like that's the way to go now. I'm updating the docs accordingly.